collectors/0/smart_stats.py

```diff
@@ -204,35 +215,53 @@ def main():
     collection_interval=config['collection_interval']
 
   # Get the list of block devices.
-  drives = [dev[5:] for dev in glob.glob("/dev/[hs]d[a-z]")]
-  # Try FreeBSD drives if no block devices found
```

Comment:
Could you please keep the FreeBSD comment so that we know where the `/dev/da` devices come from ?
Thank you 👍